settings.gradle

```diff
@@ -50,6 +51,7 @@ includeWithFlags ':site'
 
 // Examples
 includeWithFlags ':examples:annotated-http-service', 'java'
+includeWithFlags ':examples:dropwizard',             'java'
```

Comment:
I would prefer the example module was `:examples:dropwizard`, but I had to rename the folder to `dropwizard-bundle` avoid a circular dependency on `project(':dropwizard')` for some reason?

I assume that the gradle module and folder name need to match, though

Reply:
It seems like it's a known issue in Gradle. You could name the project as `:examples:dropwizard-example` and specify the project directory explicitly. In `/settings.gradle`:

```java
includeWithFlags ':examples:dropwizard-example' 'java'
project(':examples:dropwizard-example').projectDir = file('examples/dropwizard')
```

See https://github.com/gradle/gradle/issues/847#issuecomment-450259535 for more information.